distributed/tests/test_worker.py

```diff
@@ -55,6 +56,109 @@ def test_health():
         assert 'network-send' in d
 
 
+def test_worker_bad_args(loop, capsys):
+    @gen.coroutine
+    def f(c, a, b):
```

Comment:
You may be interested in switching this out for a slightly newer testing style

```python
@gen_cluster
def test_worker_bad_str_args(c, a, b):
    ...
```

See test_executor.py for examples.  I should probably rewrite the tests in this file in the new style to avoid issues like this.

Feel free to ignore.  I can handle this later.